package.json

```diff
@@ -128,52 +131,51 @@
     "@types/uuid": "^9.0.0",
     "@typescript-eslint/eslint-plugin": "^6.0.0",
     "@typescript-eslint/parser": "^6.0.0",
-    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "changesets-release-it-plugin": "^0.1.1",
-    "ember-cli": "~4.8.0",
+    "concurrently": "^8.0.1",
+    "ember-cli": "~4.12.0",
     "ember-cli-autoprefixer": "^2.0.0",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^11.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
     "ember-load-initializers": "^2.1.2",
     "ember-modifier": "^3.2.7",
     "ember-page-title": "^8.0.0",
-    "ember-qunit": "^6.0.0",
+    "ember-qunit": "^6.2.0",
     "ember-resolver": "^11.0.1",
-    "ember-source": "~4.8.4",
+    "ember-source": "~4.12.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.3.3",
+    "ember-template-lint": "^5.7.2",
     "ember-test-selectors": "^6.0.0",
     "ember-try": "^3.0.0",
-    "eslint": "^8.24.0",
+    "eslint": "^8.37.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-deprecation": "^1.4.1",
-    "eslint-plugin-ember": "^11.1.0",
-    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-ember": "^11.5.0",
+    "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-qunit": "^8.0.0",
     "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
     "prettier": "^3.0.0",
     "prosemirror-dev-tools": "^4.0.0",
-    "qunit": "^2.19.2",
+    "qunit": "^2.19.4",
     "qunit-dom": "^2.0.0",
     "release-it": "^16.1.0",
     "sass": "^1.49.9",
     "sinon": "^15.0.1",
     "typescript": "~5.2.2",
-    "webpack": "^5.74.0"
+    "webpack": "^5.78.0"
   },
   "peerDependencies": {
     "@appuniversum/ember-appuniversum": "^2.4.2",
     "@glint/template": "^1.1.0",
     "ember-cli-sass": "^11.0.1",
-    "ember-modifier": "^3.2.7"
+    "ember-modifier": "^3.2.7",
+    "ember-source": "^4.0.0"
```

Comment:
this should be 3.28 || 4 no? or is there something that breaks 3.28 support?

Reply:
Yes indeed, ember-cli-update added this.